src/pages/patientView/PatientViewPageUtils.tsx

```diff
@@ -51,3 +64,57 @@ export function getSamplesProfiledStatus(
         notProfiledIds,
     };
 }
+export function retrieveMutationalSignatureMap(
+    inputMetaData: GenericAssayMeta[]
+): MutationalSignatureLabelMap[] {
+    const mappedData = inputMetaData.map((metaData: GenericAssayMeta) => {
+        const nameSig: string =
+            'MUTATION_TYPE' in metaData.genericEntityMetaProperties
+                ? metaData.genericEntityMetaProperties['MUTATION_TYPE']
+                : '';
+        const classSig: string =
+            'MUTATION_CLASS' in metaData.genericEntityMetaProperties
+                ? metaData.genericEntityMetaProperties['MUTATION_CLASS']
+                : '';
+        const mutNameSig: string =
+            'NAME' in metaData.genericEntityMetaProperties
+                ? metaData.genericEntityMetaProperties['NAME']
+                : '';
```

Comment:
There is a util function that can be used to replace this kind of logic (get property from GenericAssayMeta): https://github.com/cBioPortal/cbioportal-frontend/blob/master/src/shared/lib/GenericAssayUtils/GenericAssayCommonUtils.ts#LL277C17-L277C53

Reply:
Missed that function, thank you! I have replaced the logic with the following (for each property):
` const name:string=getGenericAssayMetaPropertyOrDefault(metaData,'NAME','')`